.github/workflows/sonar_poc_cli.yml

```diff
@@ -44,6 +44,7 @@ on:
     types:
       - 'opened'
       - 'reopened'
+      - 'synchronize'

```

Comment:
Hi @elsegev what's the conclusion here? we'd like to see if we can conclude this PR soon.

Reply:
For each run there is a new TF workspace created in the yml file, so the jobs can run in parallel when we are pushing a new code to our branch, for every minor change.
In PR it is the same trigger, so it will not create a new workspace, but will work on the same one, and because the PR is after we are running the jobs in our branch in general, we don't want to run it for each minor change after the PR created.

Reply:
I was told that I need to re-run the build after I update my PR. Is this not correct?

Reply:
Hi Chris, I don't think you need to do that. What I do is create a PR with my initial code. At this time a github action runs and I see if I passed or not. Then I do commit, commit, commit and when I think I'm ready I manually run the github action. I don't need it to run for each commit. I prefer the freedom that I have today to not worry about triggering too many heavy actions when I do a small commit. (By commit I also mean push)

Reply:
I still don't understand, do you often push individual commits to an active PR that should not be reviewed or tested? In that case, wouldn't you want to change the PR to draft status so everyone knows not to review it, and so the actions are not run?

With the current process, it means that any outside collaborators (like me) need to learn a special process of manual triggering just for dsfkit, and then remember to do it when working on this project. It is an extra manual process I would like to avoid, as it is likely to add more back and forth in the future.

Reply:
Yes, we often push individual commits to an active PR that should not be tested because testing is a "heavy" operation.
Reviews is a different topic.